apiv2/db_import/importer.py

```diff
@@ -0,0 +1,225 @@
+    options.append(click.option("--import-tiltseries", is_flag=True, default=False))
+    options.append(click.option("--import-tomograms", is_flag=True, default=False))
+    options.append(click.option("--import-tomogram-authors", is_flag=True, default=False))
+    options.append(click.option("--import-tomogram-voxel-spacing", is_flag=True, default=False))
+    options.append(click.option("--import-everything", is_flag=True, default=False))
+    options.append(click.option("--deposition-id", type=str, default=None, multiple=True))
+    options.append(
+        click.option(
+            "--anonymous",
+            is_flag=True,
+            required=True,
+            default=False,
+            type=bool,
+            help="Use anonymous access to S3",
+        ),
+    )
+    for option in options:
+        func = option(func)
+    return func
+
+
+@cli.command()
+@click.argument("s3_bucket", required=True, type=str)
+@click.argument("https_prefix", required=True, type=str)
+@click.option("--postgres_url", required=False, type=str)
+@click.option("--filter-dataset", type=str, default=None, multiple=True)
+@click.option("--s3-prefix", required=True, default="", type=str)
+@click.option("--endpoint-url", type=str, default=None)
+@db_import_options
+def load(
+    s3_bucket: str,
+    https_prefix: str,
+    postgres_url: str,
+    s3_prefix: str,
+    anonymous: bool,
+    filter_dataset: list[str],
+    import_annotations: bool,
+    import_annotation_authors: bool,
+    import_dataset_authors: bool,
+    import_dataset_funding: bool,
+    import_depositions: bool,
+    import_runs: bool,
+    import_tiltseries: bool,
+    import_tomograms: bool,
+    import_tomogram_authors: bool,
+    import_tomogram_voxel_spacing: bool,
+    import_everything: bool,
+    deposition_id: list[str],
+    endpoint_url: str,
+):
+    load_func( s3_bucket, https_prefix, postgres_url, s3_prefix, anonymous, filter_dataset, import_annotations, import_annotation_authors, import_dataset_authors, import_dataset_funding, import_depositions, import_runs, import_tiltseries, import_tomograms, import_tomogram_authors, import_tomogram_voxel_spacing, import_everything, deposition_id, endpoint_url)
+
+def load_func(
+    s3_bucket: str,
+    https_prefix: str,
+    postgres_url: str | None = None,
+    s3_prefix: str | None = None,
+    anonymous: bool = False,
+    filter_dataset: list[str] | None = None,
+    import_annotations: bool = False,
+    import_annotation_authors: bool = False,
+    import_dataset_authors: bool = False,
+    import_dataset_funding: bool = False,
+    import_depositions: bool = False,
+    import_runs: bool = False,
+    import_tiltseries: bool = False,
+    import_tomograms: bool = False,
+    import_tomogram_authors: bool = False,
+    import_tomogram_voxel_spacing: bool = False,
+    import_everything: bool = False,
+    deposition_id: list[str] | None = None,
+    endpoint_url: str | None = None,
+):
```

Comment:
I made this a callable function so we don't have to invoke click in our tests. Things are clunky either way, but click was hiding tracebacks in our test results.